test/utils/TestConstants.sol

```diff
@@ -2,3 +2,4 @@
 pragma solidity ^0.8.25;
 
 uint32 constant TEST_DEFAULT_OWNER_FUNCTION_ID = 0;
+uint32 constant TEST_DEFAULT_VALIDATION_ID = 1;
```

Comment:
Why not reuse? Left a comment up on the stack about renaming the above from `TEST_DEFAULT_OWNER_FUNCTION_ID` to `TEST_DEFAULT_OWNER_ENTITY_ID`.